zebra-utils/README.md

```diff
@@ -24,8 +24,8 @@ To find the latest checkpoints on the `main` branch:
 1. Find the [latest completed `CI Docker` workflow run on `main`](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml?query=branch%3Amain).
    Due to GitHub UI issues, some runs will show as waiting, cancelled, or failed,
    but the checkpoints have still been generated.
-2. Go to the `Result of generate-checkpoints-mainnet` step in the
-   `Run generate-checkpoints-mainnet` job, in the `Generate checkpoints mainnet` job
+2. Go to the `Result of gen-checkpoints-mainnet` step in the
+   `Run gen-checkpoints-mainnet` job, in the `Generate checkpoints mainnet` job
 3. Scroll down until you see the list of checkpoints, it should start around line 200
 4. Add those checkpoints to the end of `zebra-consensus/src/checkpoint/main-checkpoints.txt`
 5. Repeat steps 2 to 4 for `testnet`
```

Comment:
Optional: Let's use the full job name here, to avoid confusion.

```suggestion
5. Repeat steps 2 to 4 for `Generate checkpoints testnet`
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/7643/commits/35bdfd21c1f13ee43b4aaffd8dc9608255d198ae.